index.html

```diff
@@ -50,7 +50,7 @@ <h1>
             <button class="side-panel-close" onclick="closeNav()">
```

Comment:
Would be better to use `addEventListener` here as well.

Reply:
both for `closeNav()` and `openNav()`.

Reply:
Good point.I'll fix it.